app/views/shared/_navbar.html.erb

```diff
@@ -31,6 +31,11 @@
                   </form> %>
                 </div>
             </li>
+            <li>
```

Comment:
i think this btn should display when the user signs in no?

Reply:
or is the goal to display it w/o user authentication as a means to convince the non-user to sign up to the app?

Reply:
Good point, I hadn't thought about it actually! I think the second idea is good, that way a new visitor can see right away all of the possibilities with the website (browse or make a post) but if they click on the post button, they need to make a new account/sign in. I'll add the authentication

Reply:
*Actually, the authentication should already be there right? I'm checking now

Reply:
yeah. all you need to then is just move your btn below the <% if user_signed_in? %> logic. it should under the if part, not the else

Reply:
Or maybe it should be both right? So it's present no matter is someone is signed in

Reply:
so i just checked some major marketplace websites and it was a mixed results. some websites display a listing btn while others dont. considering that, I leave the decision up to you. can talk about it tomorrow w the team as well if you're unsure about your decision.

Reply:
Let's add it for now, it can be changed back easily enough but if everyone is fine with the decision, it saves us a bit of time tomorrow